cdap-ui/app/directives/my-pipeline-configurations/my-batch-pipeline-config/my-batch-pipeline-config.html

```diff
@@ -0,0 +1,282 @@
+          <span class="fa"
+                ng-class="{'fa-caret-down': BatchPipelineConfigCtrl.showAdvanced, 'fa-caret-right': !BatchPipelineConfigCtrl.showAdvanced}">
+           </span>
+          Advanced options
+        </div>
+        <div class="advanced-options"
+              ng-show="BatchPipelineConfigCtrl.showAdvanced">
+          <div class="configuration-header"
+                ng-class="{'active': BatchPipelineConfigCtrl.activeTab === 'pipelineConfig'}"
+                ng-click="BatchPipelineConfigCtrl.activeTab = 'pipelineConfig'">
+            Pipeline Config
+          </div>
+          <div class="configuration-header"
+                ng-class="{'active': BatchPipelineConfigCtrl.activeTab === 'engineConfig'}"
+                ng-click="BatchPipelineConfigCtrl.activeTab = 'engineConfig'">
+            Engine Config
+          </div>
+          <div class="configuration-header"
+                ng-class="{'active': BatchPipelineConfigCtrl.activeTab === 'resources', 'disabled': BatchPipelineConfigCtrl.showPreviewConfig}"
+                ng-click="BatchPipelineConfigCtrl.activeTab = 'resources'">
+            Resources
+          </div>
+          <div class="configuration-header"
+                ng-class="{'active': BatchPipelineConfigCtrl.activeTab === 'alerts', 'disabled': BatchPipelineConfigCtrl.showPreviewConfig}"
+                ng-click="BatchPipelineConfigCtrl.activeTab = 'alerts'">
+            Alerts
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="configuration-content">
+
+      <div class="configuration-step-content configuration-content-container runtime-arguments"
+            ng-if="BatchPipelineConfigCtrl.activeTab === 'runtimeArgs'">
+        <div class="step-content-heading">
+          Specify Runtime Arguments or Update the Ones Derived from Preferences
+        </div>
+        <div class="runtime-arguments-labels key-value-pair-labels">
+          <span class="key-label">Name</span>
+          <span class="value-label">Value</span>
+        </div>
+        <div class="runtime-arguments-values key-value-pair-values">
+          <key-value-pairs
+            key-values="BatchPipelineConfigCtrl.runtimeArguments"
+            on-key-value-change="BatchPipelineConfigCtrl.onRuntimeArgumentsChange"
+            get-resetted-key-value="BatchPipelineConfigCtrl.getResettedRuntimeArgument"
+            resetting-fields="BatchPipelineConfigCtrl.resettingFields"
+          ></key-value-pairs>
+        </div>
+      </div>
+
+      <div class="configuration-step-content preview-config"
+            ng-if="BatchPipelineConfigCtrl.activeTab === 'previewConfig'">
+        <div class="step-content-heading">
+          Set the number of records to preview
+        </div>
+        <div class="label-with-toggle num-records-preview row">
+          <span class="toggle-label col-xs-4">Number of records to preview</span>
```

Comment:
Would be better if we can use `col-xs-5`. Could avoid the vertical scrollbar + makes the label single sentence.
<img width="838" alt="screen shot 2017-05-24 at 7 11 38 pm" src="https://cloud.githubusercontent.com/assets/1452845/26433036/e766caa4-40b4-11e7-8f19-f299277aca0b.png">